README.md

```diff
@@ -13,24 +13,30 @@ play with it and give us your feedback :)
 
 ## Installation
 
-You need to have Java (>= 11) and Node (>= 16) and npm installed.
+The recommended way to install Effekt is by running:
+```bash
+npm install -g @effekt-lang/effekt
+```
+This will download the compiler and install the `effekt` command to the directory `npm root -g`.
 
-The recommended way of installing Effekt is by running:
+You need to have Java (>= 17) and Node (>= 16) and npm installed.
```

Comment:
Sorry for the confusion, we can still keep Java 11, see here https://github.com/effekt-lang/effekt-website/pull/65#discussion_r1777069606